client/src/game/ui/reactive/StateObserver.ts

```diff
@@ -307,6 +307,12 @@ const replayObservers: Subscriptions = [
   // game start
   subAfterInit((s) => s.replay.active, replayView.onActiveChanged),
 
+  // Enter Hypothetical button
+  subAfterInit(
+    (s) => s.finished && s.replay.active,
```

Comment:
shouldn't this be `s.finished && s.replay.active && s.replay.shared === null`

Reply:
good catch. didn't test that case